asset/params.go

```diff
@@ -136,6 +136,12 @@ func (r *RegistrationParams) SetFingerprintFromDataArray(contents [][]byte) erro
 }
 
 func (r *RegistrationParams) setFingerprintFromHashes(hashes [][]byte) error {
+
+	// sort hash array by ascending order
+	if err := utils.SortBytesArray(hashes); nil != err {
```

Comment:
We can leverage the built-in sort function.
https://play.golang.org/p/q8ji51jkC-S

Reply:
Make sense. Let me update.